<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="radioButtonFileIOtestmode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>File I/O testing</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonMemoryOnlyTestMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Memory testing only</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditCmdLineBuf">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_sep_main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>95</y>
      <width>801</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRunTest">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cmd_line">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Command line</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxBuffers">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Buffer(s)</string>
    </property>
    <widget class="QCheckBox" name="checkBoxRandomBufferFill">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>(-r) fill buffer randomly</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelBufferSize">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>(-s) buffer size (MB)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditBufferSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxFiles">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>File(s)</string>
    </property>
    <widget class="QTextEdit" name="textEditFileDirName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditFileSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>71</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditFilesNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelFileDirName">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>(-t) working catalog</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelFileSize">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>(-f) file size (MB)</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelFilesNumber">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>(-n) files/buffers number</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxMemoryMode">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>381</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Memory test options</string>
    </property>
    <widget class="QCheckBox" name="checkBoxMemoryOnlyTest">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>(-e) memory only test </string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAutoMemoryTest">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(-B) auto all memory testing</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAutoMemBufSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelAutoMemBufSize">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(-b) auto buffer size (MB)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxGeneralOptions">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>229</y>
      <width>371</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>General options</string>
    </property>
    <widget class="QCheckBox" name="checkBoxLoopForever">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(-l) loop forever</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxTimeProfiling">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(-Z) time profiling</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPassesNumber">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditThreadsNumber">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelPassesNumber">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>(-p) passes number</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelThreadsNumber">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>(-j) threads number</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxVerbose">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(-v) verbose</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="action_About"/>
    <addaction name="action_Quit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="action_Help">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
